### ISSUE TITLE
Trojan virus

### ISSUE THREAD

**Issue by chasecjg** [Tatsu-syo/noMeiryoUI#113]
Why is it prompted that there is a Trojan virus?

**Comment by Tatsu-syo** [comment]
No!! Meiryo UI only changes system fonts settings via Windows API.
And, don't spy personal information. See source files.

Hmm, when I released latest version, I tested execute file by VirusTotal.
And Microsoft anti virus was OK.
What anti virus do you use?
Please send sample file (noMeiryoui.exe) to anti virus vender.
It's weird.

**Comment by hunderlinehh** [comment]
Please make sure you've downloaded from the official releases page.

**Comment by Tatsu-syo** [comment]
@hunderlinehh What vender judged as trojan?
I can't reproduce this issue without vendor name.
It's anti virus software issue.
And anti virus signature is changing every day, hour.
When I released, didn`t judged as trojan.

**Comment by hunderlinehh** [comment]
> @hunderlinehh What vender judged as trojan? I can't reproduce this issue without vendor name. It's anti virus software issue. And anti virus signature is changing every day, hour. When I released, didn`t judged as trojan.

I didn't encounter this problem. This might be someone else who uploaded an edited version of noMeiryoUI. Some people said the anti virus software is Huorong.

**Comment by Tatsu-syo** [comment]
When I upload No!! Meiryo UI, I checked archive by VirusTotal.
And, offcial upload sites are here and [My private site written in Japanese](https://www.tatsu-syo.info/index.html) and [Vector - Japanese famous upload site and uploaded in Japanese name Meiryo UI mo daikkirai!!](https://www.vector.co.jp).
If No!! Meiryo UI is judged as trojan continuously, please report here.

**Comment by Tatsu-syo** [comment]
I add warning message for antivirus software in README in repository.
